<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowUI</class>
 <widget class="QWidget" name="MainWindowUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>122</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="buttonAbout">
     <property name="text">
      <string>&amp;About...</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/qt</normaloff>:/qt</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="3">
    <widget class="QLabel" name="imager">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="buttonQuit">
     <property name="text">
      <string>&amp;Exit</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/close</normaloff>:/close</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="buttonSave">
     <property name="text">
      <string>&amp;Save</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/save</normaloff>:/save</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLineEdit" name="m_lineEdit"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
